packages/aws-cdk/test/context-providers/load-balancers.test.ts

```diff
@@ -1,258 +1,110 @@
-/* eslint-disable import/order */
-import * as cxschema from '@aws-cdk/cloud-assembly-schema';
-import * as aws from 'aws-sdk';
-import * as AWS from 'aws-sdk-mock';
-import { LoadBalancerListenerContextProviderPlugin, LoadBalancerContextProviderPlugin, tagsMatch, describeListenersByLoadBalancerArn, describeTags, describeLoadBalancers } from '../../lib/context-providers/load-balancers';
-import { MockSdkProvider } from '../util/mock-sdk';
-
-AWS.setSDK(require.resolve('aws-sdk'));
-
-const mockSDK = new MockSdkProvider();
-
-type AwsCallback<T> = (err: Error | null, val: T) => void;
-
-afterEach(done => {
-  AWS.restore();
-  done();
-});
-
-describe('utilities', () => {
```

Comment:
These tests were on custom functionality we wrote because the sdk didn't cover it for us. sdkv3 does so most of this became irrelevant. We still test the in/out of the public functions but not how we're parsing/paginating/etc because the sdk now handles that under the hood. Testing their logic is not really the place of our unit tests.